pkg/virt-controller/watch/vm.go

```diff
@@ -2904,6 +3080,22 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 			}
 		}
 	}
+
+	// Adding last-seen VM revision only once for now, since we don't yet have a revert process.
+	// Remove the hasLastSeenRevision() check to keep the last-seen CR up-to-date.
+	hasIt, err := c.hasLastSeenRevision(vm)
+	if err != nil {
+		log.Log.Object(vm).Reason(err).Error(failedGetLastSeenCRforVmErrMsg)
+		return vm, syncErr, err
+	}
+	if !hasIt {
+		_, err = c.createVMRevision(vm, revisionPrefixLastSeen)
+	}
+	if err != nil {
+		log.Log.Object(vm).Reason(err).Error(failedCreateLastSeenCRforVmErrMsg)
+		return vm, syncErr, err
+	}
```

Comment:
I still think the start controller revision is enough, actually creating the revision here is problematic because or not?